games/devtest/mods/testhud/init.lua

```diff
@@ -209,3 +209,23 @@ minetest.register_on_leaveplayer(function(player)
 	player_font_huds[player:get_player_name()] = nil
 	player_waypoints[player:get_player_name()] = nil
 end)
+
+minetest.register_chatcommand("hudprint", {
+	description = "Writes all used Lua HUD elements into chat.",
+	func = function(name, params)
+		local player = minetest.get_player_by_name(name)
+		if not player then
+			return false, "No player."
+		end
+		
```

Comment:
Tailing whitespace
*How did I notice that?*
```
.git/rebase-apply/patch:55: trailing whitespace.
		
.git/rebase-apply/patch:63: trailing whitespace.
		
.git/rebase-apply/patch:195: trailing whitespace.
	
```
in `git am -3 patchfile`

Reply:
It's fixed. Sorry, for the incontinence. Why isn't this checked by the CI/CD pipeline? o.o
My editor now automatically removes them.